src/style.css

```diff
@@ -1 +1,96 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-size: normal;
```

Comment:
The property `font-size` is incorrectly set to 'normal'. This is not a valid value for `font-size`. You should specify a size in units like `px`, `em`, `rem`, etc.